### PR TITLE
Use prettyJSON from new heroku-cli-util

### PR DESCRIPTION
**Pull request by rwz**
Use prettyJSON from `heroku-cli-util` instead of bundling the same dependencies separately.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "cardinal": "0.6.0",
```
Comment (dickeyxxx) [inline package.json]: you should take this out

Comment (rwz) [inline package.json]: you're right, i forgot

Comment (rwz) [inline package.json]: @dickeyxxx fixed

**Comment by dickeyxxx** [comment]
lgtm

**Pull request merged**